src/opening-period/page/CreateNewOpeningPeriodPage.scss

```diff
@@ -1,11 +1,5 @@
 @import 'node_modules/hds-design-tokens/lib/all.scss';
 
-.opening-period-page-header-row {
```

Comment:
Did this style definition have no effect since it is removed? :) The class seems to be still in the react component?

Reply:
Oh, I can probably remove the class also

Reply:
Fixed